### DISCUSSION TITLE
Information Architecture

### DISCUSSION THREAD

**Discussion by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#5] [Ideas]
## Overview

As part of the website redesign, I would like to rethink the information architecture of the site, mainly to:
- condense down the number of top level links
- get rid of the accordion navigation (too many clicks, relies on JavaScript)
- emphasize web standards
- better "compartmentalize" and reframe the docs, using "higher" level phrasing like _Routing_, instead of _Template and Pages_, and _Resources_ instead of _Styles and Assets_

>

## Information Architecture

**_Current_**
<details>
  <img src="https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/e37084de-71db-4018-9983-45fb8a5d2a9f">
</details>

**_Proposed_**
<details>
  <img src="https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/fcf29b68-ab97-48b1-a82c-bde243973cc5">
</details>

So from the above, some examples would be:
1. Get rid of the _About_ page.  I don't think we need to try so hard to talk about ourselves, I feel like aside from an intro paragraph in the docs, the docs should speak for themselves in this regard
1. Put _Getting Started_ into the _Guides_ section
1. Put _Plugins_ into the _Docs_ section
1. Docs get a pretty good overhaul side-navigation wise

----

Status
1. [ ] Define I.A. and navigation structures / interactions
1. [ ] Outline the docs page
1. [ ] Outline the guides page

**Comment by Auhseh** [comment]
Started doing further explorations for the Docs/Guides pages, landed on this, what do you think about this direction?

Next would be exploring different screen sizes

https://www.figma.com/file/ePhyD5qNMrBAcHwQ08BnQC/GreenWood?type=design&node-id=187%3A194&mode=design&t=zhjDSOC6IUgTuuNq-1

**Reply by thescientist13** [reply]
Yeah, this looks great!  Nice and "clean" and easy to read.  💯 

One thing that might be nice to incorporate for code snippets is adding a nice "copy to clipboard" icon / button so readers can just grab that code snippet right away.  I think you did this on the home page and would be nice to carry through to the docs / guides.  👍 

https://kit.svelte.dev/docs/routing
![Screenshot 2024-03-04 at 8 49 45 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/20292411-496d-468a-84d1-b1096aabd5c4)

**Comment by thescientist13** [comment]
Made issue to carry forward all the independent needs of completing the IA
- https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/31
- https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/33
- https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/34
- https://github.com/ProjectEvergreen/www.greenwoodjs.dev/issues/35
